integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -3,19 +3,30 @@ import {
   AxiosRequestConfig,
   AxiosResponse,
   DeepPartial,
-  Extensible,
+  EntityMap,
   HandleRequest,
   Jovo,
-  JovoError,
   NluData,
   NluPlugin,
   PluginConfig,
 } from '@jovotech/framework';
-import { RasaResponse } from './interfaces';
+import { RasaEntity, RasaIntent, RasaResponse } from './interfaces';
 
 export interface RasaNluConfig extends PluginConfig {
-  serverUrl: string;
-  serverPath: string;
+  serverUrl?: string;
+  serverPath?: string;
+  //activate alternative intent classifications in $nlu
+  //and constrain the number of alternatives by number and/or an confidence cutoff
+  alternativeIntents?: { maxAlternatives?: number; confidenceCutoff?: number };
+}
+
+interface RasaNluData extends NluData {
```

Comment:
Due to the interface not being exported, this is not used anywhere outside of `RasaNlu`. 
It should either get exported or get removed. The main issue is, that the type of `Jovo.$nlu` will always be `NluData` without casting.

Reply:
decided to export it. Plus in consequence of the other changes the return type of _process_ needs to be _RasaNluData_ now.